### ISSUE TITLE
Design nits for forms

### ISSUE THREAD

**Issue by silochad** [BeanstalkFarms/Beanstalk#361]
This is a running issue which we can update throughout Q1/Q2 with design nitpicks and fixes.

### All forms

**(1)**
![Feb-28-2023 11-05-28](https://user-images.githubusercontent.com/99763686/221925462-81d1bb7d-1ac9-4012-a933-93a4c3f509db.gif)

- [ ] (1a) Input before hover: Two different grey borders for inputs
- [ ] (1b) Input on focus: New input has green outline, old has blue
- [ ] (1c) Button before hover: 
  - The one inside the input has grey border
  - The one outside the input has a light blue border
- [ ] (1d) Button on hover: 
  - The one inside the input receives a blue background
  - The one outside the input receives a green border with light-green background

**(2)**
![image](https://user-images.githubusercontent.com/99763686/221926467-da11e0bd-5287-4aa6-9331-fb9637b1fa45.png)

- [ ] (2a) The "Balance: 100 (Max)" bit should be shifted so that the right side is aligned with the button above
- [ ] (2b) Should the balance label say "Circulating Balance", etc.? In practice, I found it a bit hard to parse the icon at a glance.

**(3)**
![image](https://user-images.githubusercontent.com/99763686/221926820-9ba8e869-3878-481a-8bc4-35fe05b887c4.png)

- [ ] (3a) Button hover: Border is blue before, green after. Confirm w/ design that this is the behavior desired.
- [ ] (3b) Some modals like this one have extra padding at the bottom. Remove the extra padding.
- [ ] (3c) Add extra padding above the buttons but below the header, so that the space from the "Destination" to the top line of the modal is the same as the space from "Destination" to the first button. (We might need to make it more like 75% of the topline distance). Other modals:
  - [ ] Farmer's Almanac

**(4)**
![Feb-28-2023 11-18-26](https://user-images.githubusercontent.com/99763686/221928937-4b557b68-f703-45c3-ae18-9f2914029629.gif)
- [ ] (4a) Believe this is switching from white -> blue border, and the blue just happens to blend in with the header background.
  - [ ] CHECK WITH DESIGN: May be fixed by theme update in Q1.

**(5)**
![Feb-28-2023 11-20-56](https://user-images.githubusercontent.com/99763686/221929124-8d3dbf2b-3a0f-4c46-9a75-879baeca3a6b.gif)
- [ ] (5a) Nav button hover background isn't legible on the blue background.
  - [ ] CHECK WITH DESIGN: May be fixed by theme update in Q1.

**(L1)** (Low Priority)
![image](https://user-images.githubusercontent.com/99763686/221927429-7d5917e7-eb1d-4cad-a720-c3be3027d278.png)
- [ ] (L1a) Border radius matches in pixel terms here, but doesn't match visually because of the padding difference. The border radius on the button needs to be reduced. Forget the algo for calculating this but it's floating around somewhere.

**(L2)** (Low Priority)
![image](https://user-images.githubusercontent.com/99763686/221929464-9ba68113-6df8-4c3b-8d8a-03e423cf4fca.png)
- [ ] (L2a) The BeaNFT icon is hard to decipher and looks strange IMO. Should we use a real image?
- [ ] (L2b) The icon colors are green while the theme is blue
- [ ] (L2c) The arrow in the dropdown line item isn't aligned with the text
- [ ] (L2d) Swap should probably be moved to a top-level nav icon or button since it's becoming increasingly important

**Comment by tansanDOTeth** [comment]
@silochad I noticed you wanted to deprecate #326, but I think we should actually prioritize it for these type of change discussions. We could incorporate something like Chromatic (https://www.chromatic.com/features/publish) to our dev and design workflow.
